examples/data-sources/fabric_precision_time/get_by_uuid.tf

```diff
@@ -0,0 +1,3 @@
+data "equinix_fabric_precision_time" "time_service" {
```

Comment:
Note that you can avoid adding a custom doc template for this datasource by moving this file to `examples/data-sources/equinix_fabric_precision_time/data-source.tf`